root/etc/e-smith/templates/etc/ssh/sshd_config/70Restricted2Sftp

```diff
@@ -0,0 +1,27 @@
+{
+    # restrict groups to sftp
+
+    use strict;
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro();
+
+    my @SftpRestricted = ();
+    foreach ( split(',',$db->get_prop('sshd','AllowGroups') || '')) {
+        my ($group, $sftp) = split(':', $_);
+        if(!$group || !$sftp) {
+            next;
+        }
+        push @SftpRestricted, $group if ($sftp eq 'SftpRestricted');
```

Comment:
Can we write it shorter? It's just a flag...

```suggestion
        push @SftpRestricted, $group if ($sftp eq 'sftp');
```

Reply:
agreed